types/selenium-webdriver/index.d.ts

```diff
@@ -2072,7 +2056,7 @@ export class WebDriver {
    * @param {!command.Executor} executor The executor to use when sending
    *     commands to the browser.
    */
-  constructor(session: Session|Promise<Session>, executor: Executor);
+  constructor(session: Session|Promise<Session>, executor: http.Executor);
```

Comment:
http.Executor could have been command.Executor... maybe for a future PR? I didn't have time to review this but I had this comment in the previous PR and it was not addressed.

Reply:
Hi @cnishina ,
Thanks for your comment!
As far as I can see http.Executor is correct, see:

https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/index.js#L641

The code is not super clear in the original selenium-webdriver package. I agree that in a future PR we could clear things up a bit more.